templates/macros/list_posts.html

```diff
@@ -11,12 +11,14 @@
                 <section class="bloglist-row bottom-divider">
             {% endif %}
 
-            {% if post.date %}
-                <div class="date">
-                    {{ macros_format_date::format_date(date=post.date, short=false, language_strings=language_strings) }}
-                </div>
-            {% endif %}
-
+            <ul class="bloglist-meta">
+                {% if post.date %}
+                <li class="date">{{ macros_format_date::format_date(date=post.date, short=false, language_strings=language_strings) }}</li>
```

Comment:
kept the `date` class here in case someone has custom CSS referencing it, but could be removed